### PR TITLE
Feature/ctech-2188-t-02002-luminesce-syntax

### PR DESCRIPTION
**Pull request by adamMQM**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb [hunk 1]
```diff
@@ -0,0 +1,3542 @@
+{
```
Comment (pms1969) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    drive_config = lusid_drive.Configuration(host="https://fbn-uni.lusid.com/drive")
</pre><p>Is it possible to configure the url at the top somewhere, so new users don't have to go looking for where to change it.  I know these don't get run on the PR requests, but if someone where to pick it up and have a go, they might find the bits earlier just work, and this bit doesn't.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (adamMQM) [reply university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: oooh I missed that - I'd bet its configurable from env vars, I'll double check and update

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>remove this and add to the image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>these actually need licenses, rename to "uk_equity_instruments" and "us_equity_instruments"</p><p> </p><p>remove references to ftse and nasdaq</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>you dont need to explicitly set the <code>effectiveAt</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    drive_config = lusid_drive.Configuration(host="https://fbn-uni.lusid.com/drive")
</pre><p> </p><p>in order to make it easy to run for users in different domains, remove these 2 lines and set the token on the <code>apiClient</code> taking it from the lusid one </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>"gonna" - maybe a bit too informal</p><p> </p><p>Sqllite &gt; SQLite</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    
</pre><p>maybe add Figi as a returned column to show they are all populated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>add Figi in the return</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>maybe explain the difference between <code>^</code> and <code>*</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p><code>Sys.</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>i think this example may benefit from adding comments explaining what each (sub)statement is doing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

**Comment by rizwansaeed** [comment]
looks good, in general can we encode the code element e.g. `SELECT` `Sys.Provider` etc

File: university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb (hunk 1 above)
Comment (nikawo) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>We use SQLite as our base SQL syntax https://www.sqlite.org/docs.html</p><p>We have then built extensively on top of this. </p><p> </p><p>Main difference to say windows SQL server is that we specify parameters in the where clause</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (nikawo) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>Do we want to explain what a provider is here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (nikawo) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>intro magic command?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb [hunk 2]
```diff
@@ -0,0 +1,4173 @@
...
```
Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>inconsistent scope</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    def upsert_portfolio(api_factory, scope, code, portfolio_name, portfolio_description):
</pre><p>remove portfolios, use instruments only</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>remove ftse refereneces</p><p> </p><p>"gonna"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <p>put in quotes</p><p> </p><p>add commentary saying we cover this over the course of a number of examples</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (rizwansaeed) [inline university/T02002-Luminesce-syntax/Luminesce Syntax.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    enduse;
</pre><p> </p><p>split out and explain (for all examples where appropriate)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Review by rizwansaeed — DISMISSED** [review]

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**